articles/architecture-scenarios/b2c/branding.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Branding
+description: How to brand Auth0 items to reflect your brand and desired user experience
+toc: true
+topics:
+    - b2c
+    - ciam
+    - branding
+    - universal-login
+    - login-pages
+    - change-password-pages
+    - custom-domains
+    - error-pages
+contentType: concept
+useCase:
+    - user-logout
+---
+# Branding
+
+Auth0 can be customized with a look and feel that aligns with your organization's brand requirements and user expectations. The consistency in appearance improves your customer's experience, giving them peace of mind that the pages they're using can be trusted and are secure.
+
+Some of the items that you can brand include:
+
+* Login, Change Password, and Error pages
+* Custom domains
+* Email templates
+
+Auth0 also supports [internationalization (I18N)](/i18n) and localization (L10N), both of which are important if you work with an international clientele.
+
+## Design considerations
+
+For some users, the out-of-box experience -- perhaps with just a touch of alteration -- is more than sufficient. However, for those wanting more control over appearance and user experience, Auth0 provides extensive flexibility when it comes to customizing and configuring things like your [Login](#universal-login), [Change Password](#change-password), and [Error](#errors) pages, [Custom Domain](#custom-domain) and [Email Templates](#email-templates). This allows you to create the user experience look and feel you need.
+
+When considering the items you want to brand, as well as how best to brand these items, you'll want to consider the following.
+
+* Will you brand your login page?
+
+* Will you need to localize your login page?
+
+* Will you customize emails so that they're not just branded, but vary based on user preference?
+
+* How will users know that they're still on your domain when they see your login page?
+
+* What do you need to do to provide additional browser security (e.g., implement Extended Validation)?
+
+* How will you direct users in the event of errors?
+
```

Comment:
Should we have a paragraph here that links the questions to the section that answers them?

Reply:
The original B2C Welcome have such paragraph(s) and I think we should retain them in some form. As per my comments in `articles/architecture-scenarios/b2c/authentication.md`, the intention behind this document is to explicitly guide the reader. The problem with "the answers to these questions and more are detailed below" - or nothing at all - is that the reader has to hunt through all the sections until they find the answer (to the questions). The verbiage in the original B2C Welcome was - rather than being specific content - actually designed to be narrative; introductory narrative if you like: the reader only has to go through one (or two) paragraphs until they find where to go next (to get the question answered). Does that make sense?